propagation.go

```diff
@@ -24,12 +19,7 @@ const (
 	FieldB = "x-instana-b-"
 )
 
-func (r *textMapPropagator) inject(spanContext ot.SpanContext, opaqueCarrier interface{}) error {
-	sc, ok := spanContext.(SpanContext)
-	if !ok {
-		return ot.ErrInvalidSpanContext
-	}
-
+func injectTraceContext(sc SpanContext, opaqueCarrier interface{}) error {
```

Comment:
The OT spec says that these methods should be named inject/extract for compatibility.  The OTel specification seems to have kept the naming: https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/context/api-propagators.md#inject

Although I like your naming scheme better, I believe this will break compatibility.

Reply:
Not really, we provide `instana.tracerS` to use as a propagator. This type satisfies [`opentracing.Tracer`](https://godoc.org/github.com/opentracing/opentracing-go#Tracer) described in the spec you've mentioned. `{inject,extract}TraceContext()` are just internal methods used by `(instana.tracerS).{Inject,Extract}()`.